### PR TITLE
Styling heading/description gallery pages

### PR DESCRIPTION
**Pull request by mirjamverloop**

### CODE REVIEW

File: components/generic/PageHeader.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<template>
+  <header class="row">
+    <b-col
+      cols="12"
+      :lg="mediaUrl ? 9 : 12"
```
Comment (mirjamverloop) [inline components/generic/PageHeader.vue:5]: Could you always set this to 9. The title and description should always stretch 9 columns.

File: components/generic/PageHeader.vue [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      </h1>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        v-if="htmlDescription"
+        v-html="htmlDescription"
+      />
+      <!-- eslint-enable vue/no-v-html -->
+      <p v-if="rawDescription">
+        {{ rawDescription }}
+      </p>
```
Comment (mirjamverloop) [inline components/generic/PageHeader.vue:21]: Does it break when you have just 1 description and use only

`<div
        v-if="htmlDescription"
        v-html="htmlDescription"
      />`
?

Comment (monkemedia) [reply components/generic/PageHeader.vue:21]: We end up with a missing <p> tag if we just use htmldescription.

File: components/generic/PageHeader.vue (hunk 1 above)
Comment (monkemedia) [reply components/generic/PageHeader.vue]: Sorry, I forgot to push my changes

File: components/generic/ContentHeader.vue [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      </h1>
+      <!-- eslint-disable vue/no-v-html -->
+      <div
+        v-if="htmlDescription"
+        v-html="htmlDescription"
+      />
+      <!-- eslint-enable vue/no-v-html -->
+      <p v-if="rawDescription">
+        {{ rawDescription }}
+      </p>
```
Comment (lbiedinger) [inline components/generic/ContentHeader.vue:21]: get rid of this and just pass 'description' in as a prop to simplify the logic here.
Passing plain text through the v-html attribute will still work with plain text.

Then in the various Pages always only pass in 'description'

File: pages/index.vue [hunk 4]
```diff
@@ -22,20 +22,11 @@
...
-          <p
-            v-if="page.headline"
-            class="lead"
-          >
-            {{ page.headline }}
-          </p>
-        </div>
-      </header>
+        :title="page.name"
+        :raw-description="page.headline"
```
Comment (lbiedinger) [inline pages/index.vue:28]: this needs to be 'description' too

Comment (monkemedia) [reply pages/index.vue]: resolved

**Pull request merged**